docs/utilities/icons.md

```diff
@@ -362,39 +362,39 @@ All icons that are currently available for use
         <p class="icon-icon-eye"></p>
       </td>
       <td>
-        <code>.icon-icon-eye</code>
+        <code>.icon-eye</code>
       </td>
     </tr>
     <tr>
       <td>
         <p class="icon-icon-eye-off"></p>
       </td>
       <td>
-        <code>.icon-icon-eye-off</code>
+        <code>.icon-eye-off</code>
       </td>
     </tr>
     <tr>
       <td>
         <p class="icon-icon-pencil"></p>
```

Comment:
I guess the same is true for all icon classes, If you check the existing classnames they don't include `.icon-icon`